tests/lwip/Makefile

```diff
@@ -1,6 +1,7 @@
-APPLICATION = lwip
-
+APPLICATION = tests_lwip
+# set default board, as native not supported yet
```

Comment:
This is not correct. It works on native. We just had weird errors when setting `BOARD` to native (see #5698).